From 4ef65907dcb1435ab8cf01242bd87231975e8bc9 Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Sun, 28 Sep 2014 20:10:45 +0300
Subject: [PATCH 1201/1202] arm: cubox-i and hummingboard defconfig

When CONFIG_JUMP_LABEL=y networking bridge code breaks.
This was observed on HummingBoard when creating a bridge between wlan0
and eth0.

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 arch/arm/configs/imx_v7_cbi_hb_defconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/configs/imx_v7_cbi_hb_defconfig b/arch/arm/configs/imx_v7_cbi_hb_defconfig
index 78d2570..164c758 100644
--- a/arch/arm/configs/imx_v7_cbi_hb_defconfig
+++ b/arch/arm/configs/imx_v7_cbi_hb_defconfig
@@ -4699,7 +4699,7 @@ CONFIG_STACK_TRACER=y
 CONFIG_KPROBES=y
 CONFIG_KPROBE_EVENT=y
 # CONFIG_KPROBES_SANITY_TEST is not set
-CONFIG_JUMP_LABEL=y
+# CONFIG_JUMP_LABEL is not set
 CONFIG_OPTPROBES=y
 
 CONFIG_HZ_1000=y
-- 
1.7.9.5

